_core_upgrader.py

```diff
@@ -253,25 +252,14 @@ def upgrade_tank(sgtk_install_root, log):
             backup_folder_name = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
             backup_path = os.path.join(core_backup_location, backup_folder_name)
             log.info("Backing up Core API: %s -> %s" % (core_install_location, backup_path))
-            
-            # first copy the content in the core folder
-            src_files = _copy_folder(log, core_install_location, backup_path)
-            
-            # now clear out the install location
-            log.info("Clearing out target location...")
-            for f in src_files:
-                try:
-                    # on windows, ensure all files are writable
-                    if sys.platform == "win32":
-                        attr = os.stat(f)[0]
-                        if (not attr & stat.S_IWRITE):
-                            # file is readonly! - turn off this attribute
-                            os.chmod(f, stat.S_IWRITE)
-                    os.remove(f)
-                    log.debug("Deleted %s" % f)
-                except Exception, e:
-                    log.error("Could not delete file %s: %s" % (f, e))
-            
+
+            # move the folder to the backup location
+            try:
+                shutil.move(core_install_location, backup_path)
```

Comment:
Switching to `shutil.move` which uses `os.rename` if the destination is on the current filesystem. Otherwise, falls back to `shutil.copy2` which copies and preserves file stats/permissions. @manneohrstrom can you think of any issues with this?